keras/src/models/model_test.py

```diff
@@ -1,14 +1,17 @@
 import os
 import pickle
 from collections import namedtuple
+from collections.abc import Callable
 
 import numpy as np
 import pytest
 from absl.testing import parameterized
 
+from keras.quantizers import GPTQConfig
```

Comment:
`from keras.src.quantizers.gptq_config import GPTQConfig`

In unit tests, we always import from `src`, otherwise you often get circular imports.

Reply:
Done